config/initializers/backtrace_silencers.rb

```diff
@@ -1,6 +1,10 @@
 # Be sure to restart your server when you modify this file.
 
 # You can add backtrace silencers for libraries that you're using but don't wish to see in your backtraces.
+
+# keep the lines from plugins in the backtrace
+Foreman::Plugin.modify_backtrace_cleaner
```

Comment:
Could we keep the silencer modifications in this file?  It seems like a more appropriate home than Foreman::Plugin itself - perhaps just keep a helper there if necessary.